pkg/packagekit/updates.less

```diff
@@ -117,26 +119,35 @@ tr.security.listing-ct-item {
 }
 
 
-/* Reformat table for narrow widths */
-@media screen and (max-width: 640px) {
+/* Hide changelog summaries in narrow widths */
+@media screen and (max-width: 80ch) {
+    .container-fluid {
+        overflow-x: hidden;
```

Comment:
The 1 line that _actually hides the horizontal scrollbar_, in cases where it might happen to show up. :wink: